src/main/resources/templates/application/client/client-summary-details.html

```diff
@@ -0,0 +1,428 @@
+<!DOCTYPE html>
+<html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'cases'},breadcrumbs=~{::#breadcrumbs})}" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>LAA CCMS - Start new application</title>
+</head>
+<body>
+<div class="govuk-width-container">
+    <div id="breadcrumbs">
+        <a href="/application/client/search" class="govuk-back-link">Cancel and return to client search</a>
+    </div>
+    <div id="main-content">
+        <form action="#" th:action="@{/application/client/details/summary}" th:object="${clientDetails}" method="post">
+            <span class="govuk-caption-l">About your client</span>
+            <h1 class="govuk-heading-l">
+                Summary of client information
+            </h1>
+            <p class="govuk-body-l">
+                You are about to create a new client record, after which you may go back and edit
```

Comment:
Is this the original text?!
"after which you may go back and edit any details that change" doesn't make sense to me.
And I presume it should be "request deletion of the record FROM the LAA"?

Reply:
The from needs changing but the rest of the text is the same

You are about to create a new client record, after which you may go back and edit any details that change, or request deletion of the record from the LAA. Please check the summary details are correct before registration.